### PR TITLE
Modifications to Essentia's SBic algorithm

### PR DESCRIPTION
**Pull request by stuartmansbridge**
... to return an array of segment values, and to bypass internal segment validation

### CODE REVIEW

File: src/algorithms/highlevel/sbic.h [hunk 1]
```diff
@@ -54,7 +56,7 @@ class SBic : public Algorithm {
     declareParameter("size2", "second pass window size [frames]", "[1,inf)", 200);
     declareParameter("inc2", "second pass increment [frames]", "[1,inf)", 20);
     declareParameter("cpw", "complexity penalty weight", "[0,inf)", 1.5);
-    declareParameter("minLength", "minimum length of a segment [frames]", "[1,inf)", 10);
+    declareParameter("minLength", "minimum length of a segment [frames]. EDIT LANDR 2016: Parameter disabled!", "[1,inf)", 10);
```
Comment (jerome-mixgenius) [inline src/algorithms/highlevel/sbic.h]: I don't understand this comment

Comment (stuartmansbridge) [inline src/algorithms/highlevel/sbic.h]: It means that whatever value you put for that parameter will be ignored. I could also remove the parameter entirely, if that would be preferable.

Comment (jerome-mixgenius) [inline src/algorithms/highlevel/sbic.h]: Yes you could remove it entirely, as well as all the commented out code. This pull request and the change tree in github is enough in my opinion.

**Comment by stuartmansbridge** [comment]
Jerome says ok

**Comment by leighsmith** [comment]
Ok

**Pull request merged**